src/libs/variables.scss

```diff
@@ -7,12 +7,14 @@ $mainLightBlue: #B9D1E0;
 
 // Fonts
 $montserrat: 'Montserrat', sans-serif;
```

Comment:
try to avoid using a specific name like `monserat` use abstract ones like `font-primary` or `font-family`  the purpose of variables is to abstract the logic.